pkg/scanner/scanner_test.go

```diff
@@ -0,0 +1,534 @@
+package scanner
+
+import (
+	"context"
+	"io/fs"
+	"testing"
+
+	"github.com/aquasecurity/defsec/pkg/providers/azure"
```

Comment:
We should not have a dependency on azure.

Reply:
yes but it's part of a multi cloud test that today lives here https://github.com/aquasecurity/trivy-aws/blob/bc16385b39ea55d6a4d789730c91d41849576d8f/pkg/scanner/scanner_test.go#L358